js-intro.js

```diff
@@ -6,105 +6,52 @@
 var mantra = "Be the dev"
 
 // 1a. Write the code that determines if there is a 'B' in mantra.
-    var mantra = "Be the dev";
-    var newMantra = mantra.includes("B");
 
-console.log(newMantra);
+
 
 // 1b. Write the code that determines if there is an 'x' in mantra.
-    var mantra = "Be the dev"
-    var newMantra2 = mantra.includes("x");
-    
-    console.log(newMantra2);
+
 
 
 // 1c. Write the code that determines if there is a 'v' in mantra.
-    var mantra = "Be the dev"
-    var newMantra3 = mantra.includes("v");
-    
-    console.log(newMantra3);
 
 
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
-var str = "Let's code"
-function checker(str){
-    if(str = str.includes("o")){
-        return "it includes the selected letter"
-    }else{
-        return "it does not include the selected letter"
-    }
-}
-console.log(checker(str));
+
+
 
 // ------------------------------ Consider the following variables:
 
 var myDog = "Snoopy"
 var myCat = "Garfield"
 
 // 2a. Create a function that returns the longer of the two variables. Expected output: "Garfield"
-var myDog = "Snoopy"
-var myCat = "Garfield"
 
-function longer(myDog, myCat){
-    if(myDog.length > myCat.length){
-        console.log("Snoopy")
-    }else if (myDog.length < myCat.length){
-        console.log ("Garfield")
-    }
-}
-longer(myDog,myCat);
 
 
-// 2b. Create a function that takes in two arguments and returns an array containing the two variables. Expected output: ["Snoopy", "Garfield"]
-var myDog = "Snoopy"
-var myCat = "Garfield"
 
-function arrAnimals(){
-    var array = (myDog.concat(" ", myCat))
-    console.log(array.split(" "))
-    
-}
-console.log(arrAnimals())
+// 2b. Create a function that takes in two arguments and returns an array containing the two variables. Expected output: ["Snoopy", "Garfield"]
 
 
 
 
 // 2c. STRETCH: Write a function that returns the two variables in all lowercase letters. Expected output: "snoopy garfield"
-var myDog = "Snoopy"
-var myCat = "Garfield"
 
-function pets(){
-    return (myDog.concat(" ", myCat))
-}
-console.log(pets().toLowerCase());
+
 
 
 // ------------------------------ Consider the following variable:
 
 var myMessage = "Learning is fun!"
 
 // 3a. Write the code that logs each letter of the message using a for loop.
-var myMessage = "Learning is fun!"
- function letter(){
-     for (var i = 0; i < myMessage.length; i++) {
-         console.log(myMessage[i])
-     }
- }
- console.log(letter())
+
 
 
 // 3b. Write the code that logs each letter of the message using map.
-var myMessage = "Learning is fun!"
 
-const message = (array) => {
-    let newMessage = []
-    for (var i = 0; i < myMessage.length; i++) {
-        newMessage.push(array[i])
-    }
-    return newMessage
-}
-console.log(myMessage.split("").map(message));
+
```

Comment:
Map will iterate over your array so you don't need a for loop if you use map.

Reply:
Yea these are one of the ones I saw Matt do with minimal code. 
So to make it work I can remove the whole loop?
const message = (array) => { 
let newMessage = []
return newMessage 
}
console.log(myMessage.split(" ").map(message)

will it be written like this?